InfoMovies/InfoMovies/Mappings/Mapping.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace InfoMovies.Mappings
+{
+    public abstract class Mapping<TSource, TDestination>
```

Comment:
Have you worked with MVVM frameworks such as Prism or MVVMCross? Do you prefer this Model / ViewModel mapping solution?  If so, why?

Reply:
I never worked with MVVM frameworks... :(
i needed to "re-discover wheel" when i learning about xamarin ... but this case makes me better developer...
I always willing to learn new technologies, frameworks, patterns and etc... :)